### DISCUSSION TITLE
Project creation error

### DISCUSSION THREAD

**Discussion by Nodoubvt** [stride3d/stride#1375] [Q&A]
Hi, I've got  VS 2022 installed and working fine ( at least with other projects ).. I've installed Stride Engine and it's dependents.  Upon creating a blank project I'm getting this popup - https://imgur.com/W3XEsnN

I've been tracking he cause all day with no luck..   Any ideas
![err](https://user-images.githubusercontent.com/98469793/159784250-f768ab3d-f314-45a8-b70c-084556765bbd.png)
?

3/23/2022 2:25:16 PM - Microsoft.VisualStudio.ExtensionManager.NotInstalledException: The extension with ID '248ff1ce-dacd-4404-947a-85e999d3c3ea' is not installed to any currently installed products.
![err2](https://user-images.githubusercontent.com/98469793/159785026-38156793-8a98-4b00-b174-5ff8af57d47a.jpg)

3/23/2022 2:56:03 PM - Searching for applicable products...
3/23/2022 2:56:03 PM - Found installed product - Global Location
3/23/2022 2:56:03 PM - Found installed product - Visual Studio Community 2022
3/23/2022 2:56:03 PM - VSIXInstaller.NoApplicableSKUsException: This extension is not installable on any currently installed products.
![err3](https://user-images.githubusercontent.com/98469793/159785699-a40ab875-f40c-4b61-8240-2ba2ac5b25d2.png)

**Comment by Nodoubvt** [comment]
Thanks.

**Comment by NicusorN5** [comment]
I think the VS plugin works with VS 2019 and below. Can anyone confirm?

Are you using a release version, or did you build Stride yourself?

**Comment by Nodoubvt** [comment]
I'm using a release copy ( non beta ), I'll try and install vs 2019,  should I remove 2022 version first ?
